code/modules/antagonists/wrestler/abilities/kick.dm

```diff
@@ -2,10 +2,7 @@
 	name = "Kick"
 	desc = "A powerful kick, sends people flying away from you. Also useful for escaping from bad situations."
 	icon_state = "Kick"
-	targeted = 1
```

Comment:
This is a massive buff to wrestling belts, it'll let them fight multiple people without having to deal with popup menus. I know them being balanced by ancient ability jank is bad but it's better than nothing.

Reply:
imo better to unjank it then balance as needed

Reply:
It says it's targeted, but its actually not. Current wrestlebelt abilities already activate on the nearest mob. Old code was overriding it, so changes here don't affect anything

Reply:
Okay I tested it and it actually uses a mouse target selector now which is less jank. Overall though if you're want to make them auto-target you're going to have to nerf wrestlebelts quite a bit since the only real counter to them is multiple people.

Reply:
I'm sorry, but I'm confused why it's being suggested that something be balanced by an anti-QoL feature??? Allowing hotkeys to be used bypasses targeting anyways, and the only time you are really going to be surrounded by multiple people is likely when you are being arrested by multiple security officers or are being beaten to death. Fighting multiple people is not a good idea.

Agree that targeting is lost now when surrounded by multiple mobs, but allowing manual targeting is not supported that well... Can look into it some more if needed but for the moment I just made it pick a random nearby mob

Reply:
Right but again in practical terms this removes the item's only counterplay. If you want to do that then you're going to have to balance it some other way.

Reply:
Okay. I'm going to undo the wrestler code for the moment, was mostly just looking to convert to hotkeys. Not sure how to best balance wrestling belt